include/nuttx/compiler.h

```diff
@@ -1110,6 +1110,14 @@
 
 #endif
 
+/* Green Hills Software definitions *****************************************/
+
+#if defined(__ghs__)
+
+#  define __extension__
```

Comment:
You could merge this PR with your other compiler.h changes: https://github.com/apache/nuttx/pull/12846